src/nvim/ex_cmds.lua

```diff
@@ -29,3274 +29,3264 @@ local FILES      = bit.bor(XFILE, EXTRA)
 local WORD1      = bit.bor(EXTRA, NOSPC)
 local FILE1      = bit.bor(FILES, NOSPC)
 
-local ADDR_LINES            =  0    -- buffer line numbers
-local ADDR_WINDOWS          =  1    -- window number
-local ADDR_ARGUMENTS        =  2    -- argument number
-local ADDR_LOADED_BUFFERS   =  3    -- buffer number of loaded buffer
-local ADDR_BUFFERS          =  4    -- buffer number
-local ADDR_TABS             =  5    -- tab page number
-local ADDR_TABS_RELATIVE    =  6    -- Tab page that only relative
-local ADDR_QUICKFIX         =  7    -- quickfix list entry number
-local ADDR_OTHER            =  99   -- something else
-
 -- The following table is described in ex_cmds_defs.h file.
 return {
   {
     command='append',
     flags=bit.bor(BANG, RANGE, ZEROR, TRLBAR, CMDWIN, MODIFY),
-    addr_type=ADDR_LINES,
+    addr_type='ADDR_LINES',
```

Comment:
What is the benefit of this? (intermediately generated .c files do not need to "look good")

Reply:
This is what https://github.com/neovim/neovim/pull/13079 did to use the typedef instead of reassigning local variables to string values.